src/includes/home/about.md

```diff
@@ -116,15 +116,15 @@ An extension is a simple collection of files that modify the browser’s appeara
 <div class="anatomy-container">
   <div class="anatomy-illustration">
     {%- include home/anatomy-v2.svg | safe -%}
-    <p class="manifest show-for-medium"><img src="/assets/img/icons/manifest.svg" alt="" />Manifest.json</p>
+    <p class="manifest show-for-medium"><img src="/assets/img/icons/manifest.svg" alt="" /><code>Manifest.json</code></p>
   </div>
   <div id="anatomy-control" class="anatomy-description">
     {%- include home/anatomy-v2-mobile.svg | safe -%}
     <button class="popup-action" data-panel="anatomy-ui"><img src="/assets/img/icons/user-interface-link.svg" alt="" />User Interface</button>
     <button class="popup-action" data-panel="anatomy-content"><img src="/assets/img/icons/content-script-link.svg" alt="" />Content Scripts</button>
     <button class="popup-action" data-panel="anatomy-background"><img src="/assets/img/icons/background-scripts-link.svg" alt="" />Background Scripts</button>
   </div>
-  <p class="manifest show-for-small-only"><img src="/assets/img/icons/manifest.svg" alt="" />Manifest.json</p>
+  <p class="manifest show-for-small-only"><img src="/assets/img/icons/manifest.svg" alt="" /><code>Manifest.json</code></p>
```

Comment:
In these two cases it should have a class to avoid changing the background and font colour:

<img width="670" alt="Firefox_Extension_Workshop___Get_help_creating___publishing_Firefox_extensions_" src="https://user-images.githubusercontent.com/1514/89520244-7a6a1580-d7d5-11ea-80ed-0c40279bc2ff.png">

Reply:
Which class?

Reply:
I’ve looked through the CSS files and I can’t find an appropriate class.  Is it okay if I add something like the following?

**In `_main.scss` (circa line 254)**:
```scss
code {
  border-radius: $global-radius;
  overflow-wrap: break-word;
}
// ↓ new class
code.nobg {
  background: none
}
```

Alternatively, I could make the `.nobg` class a standalone utility class (not attached to `code` in the styles).

How should I proceed?

Reply:
Yep that would work maybe `.nobackground` as it's a bit clearer. 

You can use nesting for this so it can be. 

```scss
code {
  border-radius: $global-radius;
  overflow-wrap: break-word;
  // ... other styles here

  .nobackground {
     background: none;
     // ... any other necessary styles here
  } 
}

Reply:
In-case you've not seen `&` before, that's a reference to the parent, in this case `&` represent `code`. As a hint you'd be able use `&` in a slightly different way to also cover the heading case in #703

<details>
    <summary>Spoiler</summary>

[Here's a very simplified example if you're stuck](https://jsfiddle.net/muffinresearch/c9b5fhL2/ )


</details>

Reply:
Cool!  I’ll update to re-sync with the `master` branch and add this class to the CSS.

Reply:
Applied class in 9a7f0f3b